site/pt/_partials/devtools/whats-new.md

```diff
@@ -6,26 +6,26 @@ Consulte a versão em inglês do <a href="/tags/new-in-devtools/" translate="no"
 
 {# ### Chrome 111 {: #chrome111 }
 
-* [Debugging HD color with the Styles pane](/pt/blog/new-in-devtools-111/#color)
-* [Enhanced breakpoint UX](/pt/blog/new-in-devtools-111/#breakpoint-redesign)
-* [Customizable Recorder shortcuts](/pt/blog/new-in-devtools-111/#recorder)
-* [Better syntax highlight for Angular](/pt/blog/new-in-devtools-111/#syntax)
-* [Reorganize caches in the Application panel](/pt/blog/new-in-devtools-111/#cache)
-* [Miscellaneous highlights](/pt/blog/new-in-devtools-111/#misc)
+* [Debugging de cores HD no painel de Estilos](/pt/blog/new-in-devtools-111/#color)
+* [Melhorias na UX dos breakpoints](/pt/blog/new-in-devtools-111/#breakpoint-redesign)
+* [Atalhos customizáveis no Gravador](/pt/blog/new-in-devtools-111/#recorder)
+* [Melhor syntax highlighting para Angular](/pt/blog/new-in-devtools-111/#syntax)
+* [Paineis de caches reorganizador na aba Aplicação](/pt/blog/new-in-devtools-111/#cache)
```

Comment:
This reads strange. It's also translated as "Reorganize caches no painel de Aplicação" later in the article, can we use that here too?

Reply:
```suggestion
* [Reorganize caches no painel de Aplicação](/pt/blog/new-in-devtools-111/#cache)
```